fract4d/c/meson.build

```diff
@@ -0,0 +1,70 @@
+fs = import('fs')
+
+jpeg = dependency('libjpeg')
+png = dependency('libpng')
+
+fract4d_stdlib = py.extension_module('fract4d_stdlib',
+    'fract_stdlib.cpp',
+
+    'model/colorutils.cpp',
+    'model/colormap.cpp',
+    'model/image.cpp',
+
+    dependencies : py_dep,
+    include_directories : ['model'],
+    cpp_args: ['-D_REENTRANT=1'],
+    install : true,
+    subdir : 'fract4d',
+)
+
+cmap_name_h = configuration_data()
+cmap_name_h.set('stdlib', fs.name(fract4d_stdlib.full_path()))
+configure_file(input : 'cmap_name.h.in',
+   output : 'cmap_name.h',
+   configuration : cmap_name_h,
+)
+
+fract4dc = py.extension_module('fract4dc',
+    'fract4dmodule.cpp',
+
+    'fract_stdlib.cpp',
+
+    'fract4dc/colormaps.cpp',
+    'fract4dc/loaders.cpp',
+    'fract4dc/sites.cpp',
+    'fract4dc/images.cpp',
+    'fract4dc/calcs.cpp',
+    'fract4dc/workers.cpp',
+    'fract4dc/functions.cpp',
+    'fract4dc/arenas.cpp',
+    'fract4dc/utils.cpp',
+    'fract4dc/calcargs.cpp',
+    'fract4dc/pysite.cpp',
+
+    'fract4dc/controllers.cpp',
+
+    'model/calcfunc.cpp',
+    'model/fractfunc.cpp',
+    'model/site.cpp',
+    'model/image.cpp',
+    'model/imagewriter.cpp',
+    'model/imagereader.cpp',
+    'model/colormap.cpp',
+    'model/worker.cpp',
+    'model/STFractWorker.cpp',
+    'model/MTFractWorker.cpp',
+    'model/pointfunc.cpp',
+    'model/stats.cpp',
+    'model/colorutils.cpp',
+
+    dependencies : [jpeg, png, py_dep],
+    include_directories : ['fract4dc', 'model'],
+    cpp_args : ['-D_REENTRANT=1', '-DTHEREADS=1', '-DPNG_ENABLED=1', '-DJPG_ENABLED=1'],
```

Comment:
typo, THEREADS should be THREADS. (I can't remember what that define actually does, there is some chance it is obsolete)

Reply:
Thanks, definitely not obsolete - that was what was causing pytest to crash!